### PR TITLE
docs: correct "--auth-method" arguments in help

### PR DESCRIPTION
**Pull request by mboersma**
**Reason for Change**:
Just a couple of typo fixes.

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

**Comment by mboersma** [comment]
I'm actually noticing several other discrepancies in the docs around this. I'll try to clean them up in one PR.

/hold

**Comment by mboersma** [comment]
Ok, there was just one other related docs fix that I found.

/hold cancel

File: docs/topics/scale.md [hunk 1]
```diff
@@ -39,5 +39,5 @@ This command will re-use the `apimodel.json` file inside the output directory as
 |--node-pool|depends|Required if there is more than one node pool. Which node pool should be scaled.|
 |--new-node-count|yes|Desired number of nodes in the node pool.|
 |--master-FQDN|depends|When scaling down a kubernetes cluster this is required. The master FDQN so that the nodes can be cordoned and drained before removal. This should be output as part of the create template or it can be found by looking at the public ip addresses in the resource group.|
-|--auth-method|no|The authentication method used. Default value is 'client_secret'. Other supported values are: 'device' and 'client_certificate'.|
+|--auth-method|no|The authentication method used. Default value is `client_secret`. Other supported values are: `cli`, `client_certificate`, and `device`.|
```
Comment (CecileRobertMichon) [inline docs/topics/scale.md]: is auth-method cli not supported?

Comment (CecileRobertMichon) [reply docs/topics/scale.md]: I think it is https://github.com/Azure/aks-engine/blob/master/cmd/root.go#L222

Comment (mboersma) [reply docs/topics/scale.md]: It is supported, but we didn't have it listed here.

Comment (CecileRobertMichon) [reply docs/topics/scale.md]: I read the diff wrong... I'll see my self out ☕️

Comment (CecileRobertMichon) [reply docs/topics/scale.md]: although thinking about it, I think it might be intentional that it's missing because it doesn't work for new users

Comment (CecileRobertMichon) [reply docs/topics/scale.md]: https://github.com/Azure/aks-engine/pull/231

Comment (CecileRobertMichon) [reply docs/topics/scale.md]: or is that device?

Comment (CecileRobertMichon) [reply docs/topics/scale.md]: nevermind, it _is_ supported

**Pull request merged**